### PR TITLE
Re-did the work done in concat-v-killing in the proper branch

### PR DESCRIPTION
**Pull request by JustKuzya**
Redid the changes in develop, that were done earlier in the master branch. 

1. Removed concatenated v{{shell-script-variable}} in Hugo-formatted MD-files
2. Updated shell-script to maintain the Type 'prototype' in addition to 'tag'm and 'branch'

### CODE REVIEW

**Comment by hiddendragonXVII** [comment]
Double checked on my machine can confirm the v is gone and builds correctly

**Comment by hiddendragonXVII** [comment]
i merged main develop and this branch with no clonflicts

**Comment by iMichaela** [comment]
@JustKuzya and @nikitawootten-nist -- Lychee fails with  https://github.com/usnistgov/OSCAL/blob/v1.1.2/src/metaschema/oscal_component-definition_metaschema.xml not found. 
We never had a file `oscal_component-definition_metaschema.xml`. We do have `oscal_component_metaschema.xml`- always had it.  This error did not exit earlier today when we reviewed the PR without the [8cec4e5](https://github.com/usnistgov/OSCAL-Reference/pull/36/commits/8cec4e51b3b90f94429f0f40a39298ddbe33da68). Can you please take a second look, please. I cannot think of any other explanation. Thank you.

File: site/archetypes/model-reference/xml-definitions.md [hunk 1]
```diff
@@ -14,7 +20,7 @@ aliases:
 {{ end }}
 ---
 
-The following is a reference for the XML element and attribute types derived from the [metaschema](https://github.com/usnistgov/OSCAL/blob/{{ getenv "HUGO_REF_BRANCH" }}/src/metaschema/oscal_{{ getenv "HUGO_MODEL_RAWNAME" }}_metaschema.xml) for this [model](https://pages.nist.gov/OSCAL/concepts/layer/{{ getenv "HUGO_LAYER_ID" }}/{{ getenv "HUGO_SCHEMA_ID" }}/).
+The following is a reference for the XML element and attribute types derived from the [metaschema](https://github.com/usnistgov/OSCAL/blob/{{ getenv "HUGO_REF_BRANCH" }}/src/metaschema/oscal_{{ getenv "HUGO_SCHEMA_ID" }}_metaschema.xml) for this [model](https://pages.nist.gov/OSCAL/concepts/layer/{{ getenv "HUGO_LAYER_ID" }}/{{ getenv "HUGO_SCHEMA_ID" }}/).
```
Comment (nikitawootten-nist) [inline site/archetypes/model-reference/xml-definitions.md]: @JustKuzya this change is causing the lychee failure (`HUGO_MODEL_RAWNAME` should be used for links to files)

Comment (nikitawootten-nist) [reply site/archetypes/model-reference/xml-definitions.md]: Resolved in 5f17a028ea5bbdb2b6be06e1a19f8a7d2268a404

**Pull request merged**